.github/workflows/python-build-and-test.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [main, master]
 
-name: package-build
+name: python-build-and-test
```

Comment:
This consolidate the `test-coverage` and `python-package` workflows, since they were a holdover from the R package and largely redundant.